### PR TITLE
add some details

### PR DESCRIPTION
**Pull request by Carreau**
This notebook re-introcuce how to run cell, it could be removed.

### CODE REVIEW

File: Day-1/IPython Kernel/Beyond Plain Python.ipynb [hunk 1]
```diff
@@ -66,7 +66,7 @@
     }
    },
    "source": [
-    "Getting help:"
+    "To get help use the question mark `?`. This will brigout the pager, use `esc` or `q` to close it with the keybard, or click on the close icon ton the top right of the pager."
```
Comment (minrk) [inline Day-1/IPython Kernel/Beyond Plain Python.ipynb]: "bring out"

File: Day-1/IPython Kernel/Beyond Plain Python.ipynb [hunk 2]
```diff
@@ -92,6 +92,13 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using two question mark will try to find the source code for the given object. "
```
Comment (minrk) [inline Day-1/IPython Kernel/Beyond Plain Python.ipynb]: marks

File: Day-1/IPython Kernel/Beyond Plain Python.ipynb [hunk 3]
```diff
@@ -104,6 +111,20 @@
...
```
Comment (minrk) [inline Day-1/IPython Kernel/Beyond Plain Python.ipynb]: marks - duplicate of cell above

Comment (Carreau) [inline Day-1/IPython Kernel/Beyond Plain Python.ipynb]: Weird... maybe a git commit -p that went wrong. Will fix.

**Pull request merged**